### PR TITLE
Feat: Added button and card components

### PR DESCRIPTION
**Pull request by vrinda-mahajan**
Kindly check for any bugs and any improvement.
Live: https://zealous-minsky-2e0ba1.netlify.app/docs.html

### CODE REVIEW

File: docs.html [hunk 1]
```diff
@@ -53,31 +53,211 @@ <h6>Fallback Avatar</h6>
...
+        <div class="m1 badge-container">
             <img class="md-avatar avatar" src="https://cdn.pixabay.com/photo/2015/10/05/22/37/blank-profile-picture-973460__480.png" />
             <span class="badge img-badge right-badge lg-badge">7</span>
         </div>
     </div>
+    <div class="container">
+        <h6>Text Badge</h6>
+        <div class="m1 badge-container">
+            <img class="lg-avatar avatar" src="https://boredhumans.b-cdn.net/faces2/738.jpg" />
+            <span class="text-badge">NEW</span>
```
Comment (Maianki) [inline docs.html]: use the alt attribute for accessibility purposes.

File: src/components/button.css [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+.secondary-btn {
+    background-color: var(--primary-color);
+    color: var(--text-color);
+}
+.secondary-btn:hover {
+    background-color: var(--btn-sec-hover-color);
+}
+/* secondary outline btn */
+.secondary-outline-btn {
+    background-color: white;
```
Comment (Maianki) [inline src/components/button.css]: Try using CSS variables in all places to maintain consistency. Everything else looks good.

**Review by VLeads — CHANGES_REQUESTED** [review]
Looks great. But also give  "cursor pointer" to buttons, icons and options

**Comment by vrinda-mahajan** [comment]
Thanks, @Maianki! I have made the changes suggested by you.

**Comment by vrinda-mahajan** [comment]
> 

Thanks for reviewing. will make the changes.

**Review by Maianki — APPROVED** [review]
Everything looks good. You can merge it.

**Pull request merged**